### PR TITLE
Ja  add cart broken

### PR DESCRIPTION
**Pull request by jeffabreu**
I manage to fix the error, but just to challenge myself I guess ruriona already did that. But here it is

### CODE REVIEW

**Comment by jeffabreu** [comment]
I just noticed I forgot to format that, I will fix that if you want

File: src/js/cart.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
 import { getLocalStorage } from "./utils.mjs";
 
 function renderCartContents() {
-  const cartItems = getLocalStorage("so-cart");
+  const cartItems = getLocalStorage("so-cart") || [];
+  if (!Array.isArray(cartItems)) {
+    console.error("Invalid cart data:", cartItems);
+    return;
+  }
```
Comment (ruriona19) [inline src/js/cart.js:8]: Thank you Jefferson, I thought it was not necessary to make any changes in this file but I was wrong.  

Actually, when you try to open the cart page and the local storage is empty then we get an error in the console, but your code takes care of it successfully

**Review by ruriona19 — APPROVED** [review]
Great Job Jefferson, I like your approach is pretty simple and handles or the scenarios in which the console error is displayed

**Review by galiciaabraham — APPROVED** [review]
Great job! I see we all got to a similar solution but definitely yours looks cleaner and is less code, I approve this branch to be merged too ;)

File: src/js/cart.js (hunk 1 above)
Comment (jeffabreu) [reply src/js/cart.js:8]: It took a time actually, but it's great to know, thanks!

**Pull request merged**